README.md

```diff
@@ -252,41 +260,60 @@ Your environment comes with preinstalled extensions. You can change which extens
 1. Let's add the `indent-rainbow` extension. To do this, go to the **extensions** list and add:
 
    ```json
-   "oderwat.indent-rainbow"
+   "oderwat.indent-rainbow" 
    ```
   
    What you did above was to add the unique identifier of an extension of the [indent-rainbow](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow&WT.mc_id=academic-79839-sagibbon). This will let Codespaces know that this extension should be pre-installed upon startup.
 
 To find the unique identifier of an extension:
 
-* Navigate to the extension's web page, like so [https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow&WT.mc_id=academic-79839-sagibbon)
+* Navigate to the extension's web page, like so [marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow&WT.mc_id=academic-79839-sagibbon)
 * Locate the _Unique Identifier_ field under **More info** section on your right side.
+
+-------
+⭐ COPILOT BONUS ⭐ 
+
+
+In `devcontainer.json`, go to the following line in the `settings` values: `"emmet.triggerExpansionOnTab": true`. Add a comma at the end of the line and press enter. See what other settings Copilot recommeneds and if they'd help you in your Codespace. 
+
    
-> 💡 Learn more here, <https://docs.github.com/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account>
+> 💡 Learn more on how to [Personalize Your GitHub Codespace](https://docs.github.com/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account)
 
+<br/>
 
 ### 2. Update to smooth scroll to a section
 
 In your site header you have links to each section below. Click one of these links and watch it scroll the page to that section. Not really a scroll, right?
 
 Let's make this a better user experience by slowing that down so the user has a sense of what is happening, and where they are navigating to on the page. 
 
-1. Open `styles.css`, which is the stylesheet for your portfolio application. We need to add a style for `html`. If you look, you'll see right now `html` and `body` styles are being set together, so let's add the following css snippet to set the scrolling for the `html` element:
+1. Open `styles.css`, which is the stylesheet for your portfolio application. We need to add a style for `html`. If you look, you'll see right now `html` and `body` styles are being set together, with no style set for `scroll-behavior`. Let's give Copilot a prompt to add this for us. 
+
+1. Below the `html` and `body` styling prompt Copilot with a comment of: 
+`/* add a smooth scroll */`
+
+1. Copilot will then suggest the CSS below:
     ```css
     html {
       scroll-behavior: smooth;
     }
     ```
+1. Press the tab key to accept the suggestion.
```

Comment:
You should add a note that the output might not be exactly correct, and if that is the case, they should continue writing until the output is close to your example